ospfd/ospf_vty.c

```diff
@@ -6350,7 +6806,19 @@ static int show_ip_ospf_database_common(struct vty *vty, struct ospf *ospf,
 				if (!ret)
 					return CMD_WARNING;
 			}
-			show_lsa_detail(vty, ospf, type, &id, &adv_router);
+			show_lsa_detail(vty, ospf, type, &id, &adv_router,
+					json);
```

Comment:
Change
show_lsa_detail(vty, ospf, type, &id, &adv_router,
					json);
to
show_lsa_detail(vty, ospf, type, &id, &adv_router,
					json_vrf);

Reply:
Done